src/main/java/bio/terra/service/dataset/flight/inheritsteward/SetAuthGcpUserRolesStep.java

```diff
@@ -11,20 +14,25 @@
 import java.util.List;
 import java.util.Objects;
 
-public record SetAuthBqJobUserStep(
+public record SetAuthGcpUserRolesStep(
     ResourceService resourceService, String custodianEmail, boolean inheritSteward)
     implements Step {
 
+  public static final List<String> SNAPSHOT_GCP_IAM_ROLES =
+      List.of(BQ_JOB_USER_ROLE, SERVICE_USAGE_CONSUMER_ROLE);
```

Comment:
Would it make sense to push this list into `ResourceService`? Or, hide it completely from the caller in a method like `ResourceService.grantSnapshotPolicies(projectId, List<String>)`?